packages/torus-sdk-ts/package.json

```
@@ -1,9 +1,9 @@
 {
-  "name": "@torus-ts/subspace",
+  "name": "@torus-network/sdk",
   "version": "0.0.0",
   "private": true,
   "type": "module",
-  "//": "@torus-ts/subspace needs to be builded and imported indirectly,",
+  "//": "@torus-network/sdk needs to be builded and imported indirectly,",
```

Biome (1.9.4)

[error] The key // was already declared.

This where a duplicated key was declared again.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)